nomos-services/chain/chain-service/src/leadership.rs

```diff
@@ -70,16 +67,26 @@ impl Leader {
                     epoch_state.total_stake()
                 );
 
-                let private_inputs = LeaderPrivate {
-                    value: utxo.note.value,
-                    note_id,
-                    sk: *self.sk.as_fr(),
-                };
+                // TODO: Get the actual witness paths and leader key
+                let aged_path = Vec::new(); // Placeholder for aged path
+                let latest_path = Vec::new();
+                let slot_secret = *self.sk.as_fr();
+                let starting_slot = 0u64; // TODO: get actual starting slot
+                let leader_pk = ed25519_dalek::VerifyingKey::from_bytes(&[0; 32]).unwrap(); // TODO: get actual leader public key
```

Comment:
I guess this will be added in this PR?

Reply:
not in this PR nor immediately. We weren't doing this before as well because we actually never checked nor constructed proofs. I'd wait for the wallet to fully support this, as there are a few things we don't support yet (witness paths in the utxo tree)